requirements/sharepoint-on-premises-user-permissions-requirements.md

```diff
@@ -27,25 +27,25 @@ Here is how you can give user these privileges:
-
-    > if((Get-PSSnapin | Where {$_.Name -eq "Microsoft.SharePoint.PowerShell"})-eq $null)
-    > {Add-PSSnapin Microsoft.SharePoint.PowerShell;}
-    > cls
-    > $username = Read-Host "Enter username";
-    > Get-SPDatabase | ForEach-Object {Add-SPShellAdmin -UserName $username -database $_.Id}
-    
+  * Type the following PowerShell command: `<Add-SPShellAdmin -UserName DOMAIN\YourAccount>` [(click here to learn more)](http://technet.microsoft.com/en-us/library/ff607596.aspx).
+  * If you want to grant PowerShell, shell access to a single database [check this article](http://technet.microsoft.com/en-us/library/ff607596.aspx) for more details.
+  * If you want to grant PowerShell shell access to all content databases run this script [download Configure-SPShellAdmin.ps1](https://www.spdockit.com/wp-content/uploads/2015/02/Configure-SPShellAdmin.zip):
+
+    if((Get-PSSnapin | Where {$_.Name -eq "Microsoft.SharePoint.PowerShell"})-eq $null)  
+    {Add-PSSnapin Microsoft.SharePoint.PowerShell;}  
+    cls  
+    $username = Read-Host "Enter username";  
+    Get-SPDatabase | ForEach-Object {Add-SPShellAdmin -UserName $username -database $_.Id}  
+
      Please note:
-   * The cmdlet Add-SPShellAdmin is going to apply to all the current SharePoint databases. If more SharePoint databases are added in the future, you might have to re-run the cmdlet again.
-   * The cmdlet might fail in some environments; please contact us for further assistance.
+      * The cmdlet Add-SPShellAdmin is going to apply to all the current SharePoint databases. If more SharePoint databases are added in the future, you might have to re-run the cmdlet again.
+      * The cmdlet might fail in some environments; please contact us for further assistance.
      
 ### SPDocKit database requirements
 
 #### Creating a new database
 
-To be able to create a new SPDocKit dedicated database, user account running the installation and configuration wizard (i.e. install account) should be granted both __dbcreator__ and __securityadmin__ role on the preferred SQL Server. This allows the account to create a new database and to assign proper privileges after creation. The install account will be automatically given db_owner privileges on the newly created database if possible. Otherwise, it is advised that the account is given that privilege manually, as it is needed for upgrading the database.
+To be able to create a new SPDocKit dedicated database, user account running the installation and configuration wizard (i.e. install account) should be granted both __dbcreator__ and __securityadmin__ role on the preferred SQL Server. This allows the account to create a new database and to assign proper privileges after creation. The install account will be automatically given __db_owner__ privileges on the newly created database if possible. Otherwise, it is advised that the account is given that privilege manually, as it is needed for upgrading the database.
```

Comment:
the user account running the installation
should be granted both __the__ __dbcreator__ and __securityadmin__ roles
SQL Server - casing, whatever you've decided should be the standard
on the newly created database, if possible -  missing comma